rules/Renaming/Rector/Name/RenameClassRector.php

```diff
@@ -80,11 +81,12 @@ public function getNodeTypes(): array
             Expression::class,
             ClassLike::class,
             Namespace_::class,
+            If_::class,
```

Comment:
this `If_` addition on `RenameClassRector` is to handle docblock auto import, something like

https://phpstan.org/r/0dfb5c2e-4ea1-4f1a-8419-4202b21f4f2b

on fixture

https://github.com/rectorphp/rector-src/blob/f0b8af88eb08be29b5fbf18f0e26283600c2c7e6/rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNamesWithoutRemoveUnusedImport/rename_docblock2.php.inc#L7-L8

we can add more `Stmt` for rename docblock when there is more use cases 👍